### PR TITLE
Improve computer handling

### PR DESCRIPTION
**Pull request by alvarolobato**
@reviewbybees esp @jnord @amuniz

### CODE REVIEW

File: src/main/java/org/jenkinsci/plugins/pipeline/maven/WithMavenStepExecution.java [hunk 1]
```diff
@@ -596,37 +597,32 @@ public void stop(Throwable cause) throws Exception {
...
-            if (computer.isOffline()) {
-                LOGGER.log(Level.FINE, "{0} is offline", node);
-                return null;
-            }
+        computer = j.getComputer(node);
+        if (computer == null) {
+            throw new AbortException("No such computer " + node);
+        }
 
+        if (computer.isOffline()) {
```
Comment (jtnord) [inline src/main/java/org/jenkinsci/plugins/pipeline/maven/WithMavenStepExecution.java]: should this abort?

if the computer is offline after the workflow has entered the `node` block then you should still be able to use it.  (if it is temporarily offline).  if it is offline as the master has lost the connection then should this abort?  pipelines are supposed to be (mostly) tolerant to slave disconnects...

Comment (alvarolobato) [inline src/main/java/org/jenkinsci/plugins/pipeline/maven/WithMavenStepExecution.java]: I guess that for this case we can not do that check... the only thing I need to have is a reference to the computer and the node... having that if the computer is offline should be handled by the pipeline framework it self. That said I'm removing that check.

**Comment by cyrille-leclerc** [comment]
🐝

**Comment by alvarolobato** [comment]
@reviewbybees done

**Pull request merged**